pyproject.toml

```diff
@@ -40,7 +40,7 @@ standard_library = ["typing_extensions"]
 
 
 [tool.ruff]
-target-version = "py38"
+target-version = "py39"
```

Comment:
This is going to trigger a lot of RUFF rules. I am worried that the old JIT code has some issue with newer type annotations like `list[], tuple[]` etc, and ruff automatically updates to these if `__future__` annotations is up to date.